### PR TITLE
Fix bug in running 'git log'

### PR DESCRIPTION
**Pull request by mikkoi**
$git->run(qw/log -1 --format='%G?'/, $commit->commit);
results in:
'letter' (with single quotation marks),
not in plain letter.

Signed-off-by: Mikko Koivunalho <mikko.koivunalho@iki.fi>

### CODE REVIEW

File: lib/Git/Hooks/CheckCommit.pm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use strict;
```
Comment (gnustavo) [inline lib/Git/Hooks/CheckCommit.pm]: Why have you deleted this line? It doesn't seem to be related to the bug you fixed below.

**Review by gnustavo — CHANGES_REQUESTED** [review]
The fix is OK. I just didn't get why you deleted the "use strict;" line.

File: lib/Git/Hooks/CheckCommit.pm (hunk 1 above)
Comment (mikkoi) [reply lib/Git/Hooks/CheckCommit.pm]: Good thing you noticed. I have no idea how that came along.

I reformulated the commit and split it into two.
